srcpkgs/dbus/template

```diff
@@ -14,8 +14,9 @@ short_desc="Message bus system"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2.0-or-later"
 homepage="https://dbus.freedesktop.org/"
+changelog="https://raw.githubusercontent.com/freedesktop/dbus/dbus-1.12/NEWS"
 distfiles="https://dbus.freedesktop.org/releases/dbus/dbus-${version}.tar.gz"
-checksum=e2dc99e7338303393b6663a98320aba6a63421bcdaaf571c8022f815e5896eb3
+checksum=4b693d24976258c3f2fa9cc33ad9288c5fbfa7a16481dbd9a8a429f7aa8cdcf7
 conf_files="/etc/dbus-1/session.conf /etc/dbus-1/system.conf"
```

Comment:
@maxice8 is there still a need to define these `conf_files=` ? I mean, they don't seem to be used anymore...
````
$ cat /etc/dbus-1/session.conf
<!--
This configuration file is no longer required and may be removed.

In older versions of dbus, this file defined the behaviour of the well-known
session bus. That behaviour is now determined by
/usr/share/dbus-1/session.conf, which should not be edited.

For local configuration changes, create a file
session-local.conf ...
````

And:
````
$ cat /etc/dbus-1/system.conf
<!--
This configuration file is no longer required and may be removed.

In older versions of dbus, this file defined the behaviour of the well-known
system bus. That behaviour is now determined by
/usr/share/dbus-1/system.conf, which should not be edited.

For local configuration changes, create a file
system-local.conf ...
````